src/lib/_imports/components/c-form.css

```diff
@@ -38,6 +38,9 @@
 
 :--form__field {
   margin-bottom: 2rem; /* mimic <p> `margin-bottom` */
+
+  display: flex;
+  flex-direction: column;
```

Comment:
All form fields, Portal or CMS, checkbox or not, should be stretched. Whether the `<input>` itslef is stretched, is managed by styles specific to certain `<input>` fields.